micropython/senml/examples/basic_cbor.py

```diff
@@ -0,0 +1,43 @@
+"""
+The MIT License (MIT)
+
+Copyright (c) 2018 KPN (Jan Bogaerts)
+Copyright (c) 2023 MicroPython
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+"""
+
+
+from senml import *
+import utime as time
+import ubinascii
+from cbor2 import decoder as cbor_decoder
+
+pack = SenmlPack("device_name")
+
+while True:
+    with SenmlRecord(
+        "test", value=10
+    ) as rec:  # use a with statement to automatically remove the item from the list when it goes out of scope, generate a value for the record
+        pack.add(rec)
+        cbor_val = pack.to_cbor()
+        print(cbor_val)
+        print(ubinascii.hexlify(cbor_val))
```

Comment:
Could use `cbor_val.hex()` (unless there is a mistmatch of str/bytes bytes here).

Reply:
@dpgeorge I have no idea, I never ran those examples, but I changed it as you suggested.